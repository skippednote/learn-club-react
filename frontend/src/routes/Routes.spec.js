import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Routes from './index';

describe('Counter', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Routes />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

  it('renders the Header', () => {
    const wrapper = shallow(<Routes />);
    const header = wrapper.find('Header');
    expect(header).toHaveLength(1);
  });

  it('renders the title', () => {
    const wrapper = shallow(<Routes />);
    const title = wrapper.find('Route');
    expect(title).toHaveLength(2);
  });
});
