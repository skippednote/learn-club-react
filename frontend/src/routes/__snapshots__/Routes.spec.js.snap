// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter matches snapshot 1`] = `
<BrowserRouter>
  <React.Fragment>
    <Header />
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/people"
    />
  </React.Fragment>
</BrowserRouter>
`;
